appveyor.yml

```diff
@@ -34,30 +34,6 @@ environment:
       PUPPET_GEM_VERSION: ~> 6.0
       RUBY_VERSION: 25-x64
       CHECK: parallel_spec
-    -
```

Comment:
have you tried the tests in appveyor ? the localhost tests for litmus

Reply:
I'm doing that now.